packages/generate-arg-types/src/index.ts

```diff
@@ -1,9 +1,15 @@
+#!/usr/bin/env -S node --experimental-specifier-resolution=node
+/* eslint-disable no-console */
+
 import path from "path";
 import { z } from "zod";
 import { withCustomConfig } from "react-docgen-typescript";
 import fg from "fast-glob";
+
 import fs from "fs-extra";
-import { propsToArgTypes } from "../src/arg-types/utils";
+import { propsToArgTypes } from "./arg-types";
+
+const GENERATED_FILES_DIR = "__generated__";
```

Comment:
this is clear enough, no need to make it also bad looking one haha
```suggestion
const GENERATED_FILES_DIR = "generated";
```

Reply:
same as default in folder for generated files in relay. Imo underscores pointing that its not usual folder like in modern folder based routers signs on files like + $ etc are used